crates/wasmtime/src/runtime/vm/gc/disabled.rs

```diff
@@ -23,6 +23,21 @@ unsafe impl GcRuntime for DisabledCollector {
     }
 
     fn layouts(&self) -> &dyn GcTypeLayouts {
+        &DisabledLayouts
+    }
+}
+
+struct DisabledLayouts;
+impl GcTypeLayouts for DisabledLayouts {
+    fn array_length_field_offset(&self) -> u32 {
+        unreachable!()
+    }
+
+    fn array_layout(&self, _: &WasmArrayType) -> GcArrayLayout {
+        unreachable!()
+    }
+
+    fn struct_layout(&self, _: &WasmStructType) -> GcStructLayout {
```

Comment:
Did this sneak in? Or was this to fix something else?

Reply:
This was due to the refactor in 8445481; in order to get rid of the duplication there we need to add the disabled version of this trait in bbce595.

Reply:
This is now gone as a result of fixing conflicts; some intervening commit started using an `Option` to solve this.